FindYourFriends/screens/HomeScreen.js

```diff
@@ -32,28 +34,33 @@ const HomeScreen = (props) => {
 };
 
 const UserInfo = ({ setTest }) => {
-  //const [username, setUsername] = useState("");
-  // This is keeping state for us. Just look at truthy values on these 3, and its fine.
-  const [User, { loading, error, data }] = useLazyQuery(facade.GET_USER); // https://www.apollographql.com/docs/react/api/react-hooks/#uselazyquery
+  // This is keeping state for us.
+  // https://www.apollographql.com/docs/react/api/react-hooks/#uselazyquery
+  const [User, { loading, error, data, called }] = useLazyQuery(
+    facade.GET_USER,
+    {
+      fetchPolicy: "network-only",
```

Comment:
Otherwise it is using cache, and then when you refetch, you think it hasnt logged you properly out (by deleting token etc), when it actually has, and is just getting from cache, and not backend.